torchtune/data/_prompt_templates.py

```diff
@@ -152,3 +152,18 @@ def __call__(self, messages: List[Message]) -> List[Message]:
 
 Please see :class:`~torchtune.data.PromptTemplate` for full API arguments.
 """
+QuestionAnswerTemplate = partial(
```

Comment:
These do render awkwardly in the live docs, just fyi: 

<img width="866" alt="Screenshot 2024-08-08 at 7 24 58 AM" src="https://github.com/user-attachments/assets/6a167b5d-312d-46a0-b6be-844963cdd9a9">

Not sure if there's a way around this but just something to be aware of

Reply:
so it renders correctly if I did a partial of a function, but not a partial of a class :(